scripts/production/batch_lastz.pl

```diff
@@ -149,12 +153,68 @@
 
 # Get the division from the given master database
 my $division = ucfirst($dba->get_division());
-$division =~ s/Vertebrates/Ensembl/;
-
-print "\nPipeline commands:\n------------------\n";
+$division =~ s/Grch37/GRCh37/;
+$division =~ s/Citest/CITest/;
+my $division_path = $division =~ s/Vertebrates/Ensembl/r;
+
+# Load all the elements necessary to create one JIRA ticket per batch:
+# Initialize logger
+my $logger = Bio::EnsEMBL::Utils::Logger->new();
+# Initialize parameters used to create the JIRA tickets
+my $user = Bio::EnsEMBL::Compara::Utils::JIRA->validate_user_name( $ENV{'USER'}, $logger );
```

Comment:
You moved a lot of this sort of stuff to be "behind the scenes" in your updated `Utils::JIRA` class - do you plan to use your new system in this script?

Reply:
I'm working at the moment in all the scripts that use `Utils::JIRA` so that they take advantage of the new class too. I'll do a commit per script :)